### ISSUE TITLE
logging to azure storage

### ISSUE THREAD

**Issue by guyta-essence** [rancher/dashboard#2676]
can not create logging to azure storage

**Comment by gaktive** [comment]
Do you have more details about what errors or behaviour you're seeing when you try to do this? Which version of Rancher are you using?

**Comment by guyta-essence** [comment]
yes, [documentation ](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/plugins/outputs/azurestore/) states that to use azure blob as output you need to add:

1. azure_storage_account
2. azure_storage_access_key
3. azure_storage_sas_token
bu there is no place to input the azure_storage_sas_token
![image](https://user-images.githubusercontent.com/58657364/115348683-9a9f0080-a1bb-11eb-9cfe-02b63f6a6b9c.png)

**Comment by nwmac** [comment]
This is an old issue. Marking as stale. Please re-test with the latest Rancher and re-open if this is still an issue you are facing.
